templates/settings.yml.erb

```diff
@@ -71,12 +73,13 @@
 # https is enabled if certificate, CA certificate, and private key are present in locations specifed by
 # ssl_certificate, ssl_ca_file, and ssl_private_key correspondingly
 # default values for https_port is 8443
-<% if ssl = scope.lookupvar("foreman_proxy::ssl") -%>
-#:http_port: 8000
-:https_port: <%= scope.lookupvar("foreman_proxy::port") %>
-<% else -%>
-#:https_port: 8443
-:http_port: <%= scope.lookupvar("foreman_proxy::port") %>
+<% if ssl -%>
+<%= "#:http_port: 8080\n" if !http -%>
```

Comment:
What I was thinking was:
```erb
<% if ssl -%>
:https_port: <%= scope.lookupvar("foreman_proxy::ssl_port") %>
<% else -%>
#:https_port: 8443
<% end -%>
<% if http -%>
:http_port: <%= scope.lookupvar("foreman_proxy::http_port") %>
<% else -%>
#:http_port: 8080
<% end -%>
```

IMHO it's not weird because you get a consistent order of parameters. It's also more readable code.